src/components/ApproveRequests.vue

```diff
@@ -1,44 +1,18 @@
 <template>
-    <div class="user-campaigns" v-for="campaign in userContributions" v-bind:key="campaign.id">
-      <section class="campaign-card">
-        <section>
-        <div class="img-div"><img :src=campaign.image /></div>
-        </section>
-        <div>
-          <div class="campaign-info">
-            <h1> {{campaign.name}}</h1>
-            <p>{{campaign.description}}</p>
-          </div>
-          <div class="requests"><b>Total Request:</b> {{campaign.requests.length}}</div>
-            <div class="requests" v-for="request in campaign.requests" v-bind:key="request.id">
-            <div class="">
-              Description: {{request.description}}
-            </div>
-            <div class="">
-              Value: {{request.value}}
-            </div>
-            <div class="">
-              <button type="button" @click="approvalCount(campaign.address, request.eth_id)">
-              approvalCount</button>
-            </div>
-            <button @click="approveRequest(campaign.address, request.eth_id, request.id)">
-              Approve Request</button>
-          </div>
-        </div>
-      </section>
-      <router-link
-            style="text-decoration: none;"
-            :to="{ name: 'Campaign Request',
-            params: {id: campaign.id, address: campaign.address}}">
-      <button>Create A Request</button>
-      </router-link>
-    </div>
+    <ul class="campaigns" v-for="campaign in userContributions" v-bind:key="campaign.id">
+      <request-list
+        v-bind:campaign="campaign"
+        v-on:approval-count="approvalCount"
+        v-on:approve-request="approveRequest"
+      />
+    </ul>
```

Comment:
This lives in a separate component now. Will work on making it modular enough to input other methods/props.